homeassistant/components/synology_dsm/strings.json

```diff
@@ -30,14 +30,14 @@
       }
     },
     "error": {
-      "connection": "Connection error: please check your host, port & ssl",
-      "login": "Login error: please check your username & password",
+      "cannot_connect": "[%key:common::config_flow::error::cannot_connect%]",
+      "invalid_auth": "[%key:common::config_flow::error::invalid_auth%]",
       "missing_data": "Missing data: please retry later or an other configuration",
       "otp_failed": "Two-step authentication failed, retry with a new pass code",
-      "unknown": "Unknown error: please check logs to get more details"
+      "unknown": "[%key:common::config_flow::error::unknown%]"
     },
     "abort": {
-      "already_configured": "Host already configured"
+      "already_configured": "[%key:common::config_flow::abort::already_configured_device%]"
```

Comment:
@MartinHjelmare What should we do with Localise keys ?
Delete old ?
Rename old to new ?

Reply:
`already_configured` needs to stay since it's used by our unique_id abort helper in config entries api.

Reply:
And `cannot_connect` and `invalid_auth` ?

Reply:
They are not used by helpers. We want those keys to use the standard key names (`cannot_connect`, `invalid_auth`).

Reply:
We're not adding new keys here, so we shouldn't need to change anything in Lokalise.